testing/trino-faulttolerant-tests/src/test/java/io/trino/faulttolerant/mongodb/BaseMongoFailureRecoveryTest.java

```diff
@@ -0,0 +1,135 @@
+            throws Exception
+    {
+        return createMongoQueryRunner(
+                closeAfterClass(new MongoServer()),
+                configProperties,
+                coordinatorProperties,
+                Map.of(),
+                requiredTpchTables,
+                runner -> {
+                    runner.installPlugin(new FileSystemExchangePlugin());
+                    runner.loadExchangeManager("filesystem", ImmutableMap.of(
+                            "exchange.base-directories", System.getProperty("java.io.tmpdir") + "/trino-local-file-system-exchange-manager"));
+                });
+    }
+
+    @Override
+    protected void createPartitionedLineitemTable(String tableName, List<String> columns, String partitionColumn)
+    {
+    }
+
+    @Override
+    public void testJoinDynamicFilteringDisabled()
+    {
+        assertThatThrownBy(super::testJoinDynamicFilteringDisabled)
+                .hasMessageContaining("Unknown session property mongodb.dynamic_filtering_wait_timeout");
+        throw new SkipException("skipped");
+    }
+
+    @Override
+    public void testJoinDynamicFilteringEnabled()
+    {
+        assertThatThrownBy(super::testJoinDynamicFilteringEnabled)
+                .hasMessageContaining("Unknown session property mongodb.dynamic_filtering_wait_timeout");
+        throw new SkipException("skipped");
+    }
+
+    @Override
+    public void testAnalyzeTable()
+    {
+        assertThatThrownBy(super::testAnalyzeTable).hasMessageMatching("This connector does not support analyze");
+        throw new SkipException("skipped");
+    }
+
+    @Override
+    public void testDelete()
+    {
+        assertThatThrownBy(super::testDeleteWithSubquery).hasMessageContaining("Unsupported delete");
+        throw new SkipException("skipped");
+    }
+
+    @Override
+    public void testDeleteWithSubquery()
+    {
+        assertThatThrownBy(super::testDeleteWithSubquery).hasMessageContaining("Unsupported delete");
+        throw new SkipException("skipped");
+    }
+
+    @Override
+    public void testMerge()
+    {
+        assertThatThrownBy(super::testMerge).hasMessageContaining("This connector does not support merges");
+        throw new SkipException("skipped");
+    }
+
+    @Override
+    public void testRefreshMaterializedView()
+    {
+        assertThatThrownBy(super::testRefreshMaterializedView)
+                .hasMessageContaining("This connector does not support creating materialized views");
+        throw new SkipException("skipped");
+    }
+
+    @Override
+    public void testUpdate()
+    {
+        assertThatThrownBy(super::testUpdate).hasMessageContaining("This connector does not support updates");
+        throw new SkipException("skipped");
+    }
+
+    @Override
+    public void testUpdateWithSubquery()
+    {
+        assertThatThrownBy(super::testUpdateWithSubquery).hasMessageContaining("This connector does not support updates");
+        throw new SkipException("skipped");
```

Comment:
CI is red. These failure messages should be updated. 
```
Error:    TestMongoTaskFailureRecoveryTest>BaseMongoFailureRecoveryTest.testUpdateWithSubquery:126 
Expecting throwable message:
  <"This connector does not support modifying table rows">
to contain:
  <"This connector does not support updates">
but did not.
```

Reply:
Just pushed an update with this and the other few fixed. 🤞